config/owasp/suppressions.xml

```diff
@@ -490,22 +487,56 @@
         <vulnerabilityName>Disallow calling helperMissing and blockHelperMissing directly</vulnerabilityName>
         <vulnerabilityName>Prototype pollution</vulnerabilityName>
     </suppress>
-
-
     <suppress>
         <notes><![CDATA[
    file name: jruby-complete-1.7.26.jar: readline.jar
    ]]></notes>
         <packageUrl regex="true">^pkg:maven/org\.jruby/readline@.*$</packageUrl>
         <cpe>cpe:/a:jruby:jruby</cpe>
     </suppress>
-
     <suppress>
         <notes><![CDATA[
    file name: jruby-complete-1.7.26.jar: readline.jar
    ]]></notes>
         <filePath regex="true">.*\bbcprov-jdk15on-1.54\.jar</filePath>
         <cpe>cpe:/a:bouncycastle:legion-of-the-bouncy-castle-java-crytography-api</cpe>
     </suppress>
-
+    <suppress>
+        <notes><![CDATA[
+    file name: ant*.jar
+    ]]></notes>
+        <filePath regex="true">.*\**\.jar</filePath>
+        <cpe>cpe:/a:apache:ant</cpe>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: camel*.jar
+   ]]></notes>
+        <filePath regex="true">.*\**\.jar</filePath>
```

Comment:
I bumped up camel version in my PR. we can take that here. no need of camel supressions. only ant is needed

Reply:
Addressed in this commit: https://github.com/hmcts/rd-judicial-data-load/pull/122/commits/4b651cec9f3b09500a8de41cbb8ff674edc5cf1f